### PR TITLE
Add changelog for the v0.11.0 beta releases

### PR DESCRIPTION
**Pull request by xmudrii**
**What this PR does / why we need it**:

This PR adds changelogs for v0.11.0-beta.1, v0.11.0-beta.2, and v0.11.0-beta.3 releases.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @xrstf

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+# [v0.11.0-beta.3](https://github.com/kubermatic/kubeone/releases/tag/v0.11.0-beta.0) - 2019-12-20
+
+## Attention Needed
+
+* Kubernetes 1.14 clusters are not supported as of this release because 1.14 isn't supported by the upstream anymore
+  * It remains possible to upgrade 1.14 clusters to 1.15 and is strongly advised
```
Comment (xrstf) [inline CHANGELOG.md]: Sounds IMHO better as `It remains possible and is advisable to upgrade ...`.

Comment (xmudrii) [reply CHANGELOG.md]: Done

**Pull request merged**